integr8ly/integreatly-operator
# INTLY-3286: Add installed_services param to the webapp dc

## What
Adds an `INSTALLED_SERVICES` param that contains the installed products information i.e. dashboard url, version. This ensures that the webapp has access to the product routes so that it can add these links to the *Applications* section in the solution explorer's landing page.
- [x] Add `INSTALLED_SERVICES` param to the webapp CR.
- [x] Move the solution explorer installation to the very last stage.
    - This should ensure that all the products have been successfully installed first
- [x] Bump operator version

## Verification Steps
1. Install the Integreatly operator and create an installation CR.
   - Install using an operator source pointing to `jameelb` app registry or run locally
2. **Verify**: All products installed by Integreatly should be completed successfully.
3. **Verify**: Solution Explorer should be installed as the last stage
   - Check the installation CR and ensure that there is a stage called `solutionexplorer`.
   - Ensure that the phase of the solution explorer is `completed` with the correct host and version.
4. **Verify**: `INSTALLED_SERVICES` param is specified in the webapp CR in the solution explorer namespace.
   - Ensure that it doesn't contain AMQ online
   - Ensure that it doesn't contain products that doesn't have a dashboard URL.
   - Ensure that the product names matches the [product id](https://github.com/integr8ly/tutorial-web-app/blob/master/src/product-info.js) that the webapp expects it to be.
